src/components/button/button.spec.ts

```diff
@@ -10,15 +10,35 @@ describe('Button', () => {
 
   const textVariable = 'Varnish'
 
+  it('emits click event when clicked', async () => {
+    const wrapper = mount(Button);
+    await wrapper.trigger('click');
+    expect(wrapper.emitted('click')).toHaveLength(1);
```

Comment:
For good measure, you can also check before the click, that emitted('click') don't have a length.

Reply:
not sure why but expect(wrapper.emitted('click')).toHaveLength(0);
fails the test
same with 
expect(wrapper.emitted('click')).not.toHaveLength(1);